lab_1_classify_by_unigrams/main.py

```diff
@@ -28,6 +50,14 @@ def create_language_profile(language: str, text: str) -> dict[str, str | dict[st
     :param text: a text
     :return: a dictionary with two keys – name, freq
     """
+    if not isinstance(language, str) or not isinstance(text, str):
+        return None
+
+    tokens = tokenize(text)
```

Comment:
you can set the value to the dict without creating a variable

Reply:
fixed